### PR TITLE
Add com.github.sqlitebrowser.sqlitebrowser

### PR DESCRIPTION
**Pull request by Qwertie-**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
  - `<content_rating>` and `<release>` are missing from appdata in the tagged release. Its in master but not tagged yet. Is this a problem?
- [x] My pull request follows the instructions at [App Submission][submission].
- [ ] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
  - `--filesystem=home` The app does sort of support portals but there are errors after opening files. FS access makes sense until that can be resolved
  - `--share=network` The only place I can see this used is for printing. Removing network access only allows the "Print to file" option. I guess if people want to lock this down more they can use flatseal but better to have the default config with everything working.
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [ ] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link: https://github.com/sqlitebrowser/sqlitebrowser/issues/2103**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
  - Its a github repo I do not control
- [ ] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*
  - No patches used

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by nedrichards** [comment]
@Qwertie- you'll need to take the appdata changes as a patch otherwise I suspect it'll fail the validator but we can find out.

**Comment by nedrichards** [comment]
bot, build com.github.sqlitebrowser.sqlitebrowser

**Comment by Qwertie-** [comment]
Looks like the spec couldn't find the appdata file at all. So its possible to copy the app data in to this branch to be used instead of the one in the main repo?

File: com.github.sqlitebrowser.sqlitebrowser.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+app-id: com.github.sqlitebrowser.sqlitebrowser
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: sqlitebrowser
+rename-icon: sqlitebrowser
+rename-appdata-file: sqlitebrowser.appdata.xml
```
Comment (bilelmoussaoui) [inline com.github.sqlitebrowser.sqlitebrowser.yaml:7]:
```suggestion
rename-appdata-file: sqlitebrowser.desktop.appdata.xml
```
That's the correct file name upstream, you probably want to rename those upstream

**Comment by barthalion** [comment]
bot, build com.github.sqlitebrowser.sqlitebrowser

**Comment by barthalion** [comment]
Appdata needs some fixes:

```
builddir/files/share/appdata/com.github.sqlitebrowser.sqlitebrowser.appdata.xml: FAILED:
• tag-missing           : <content_rating> required [use https://odrs.gnome.org/oars]
• tag-missing           : <release> required
```

**Comment by Qwertie-** [comment]
@barthalion Those fixes are in master but not in the tagged release. I don't control the upstream repo so I can't fix this. Is it possible we can include the latest appdata file in this PR

**Comment by nedrichards** [comment]
@Qwertie- in that case you should ideally apply the upstream change as a patch and then drop it once upstream makes a new release.

**Comment by barthalion** [comment]
@Qwertie- Yeah, ideally just apply files here "downstream" for now. Also DRI permission is fine.

**Comment by hfiguiere** [comment]
A patch for the appdata would make this app closer to submission.

**Comment by Qwertie-** [comment]
I'm not certain how to patch the file but I'll give it a shot looking at some of the other pull requests as a guide.

File: com.github.sqlitebrowser.sqlitebrowser.desktop.appdata.xml [hunk 2]
```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>sqlitebrowser.desktop</id>
```
Comment (hfiguiere) [inline com.github.sqlitebrowser.sqlitebrowser.desktop.appdata.xml:3]: the id is wrong. (it was handled previously by the `rename-appdata-file`)

**Comment by hfiguiere** [comment]
bot, build com.github.sqlitebrowser.sqlitebrowser

**Comment by hfiguiere** [comment]
The appdata isn't referenced in sources, nor installed. Patching it would be much better. Since you already have a Pull Request with upstream, the hardest is already done with a creating a patch. It's just a matter of adding the source patch entry.

**Comment by Qwertie-** [comment]
@hfiguiere I'm not quite sure what you are suggesting here? Do you mean leaving the app data filename as it is and keeping the yml config to rename it?

**Comment by hfiguiere** [comment]
As said previously by @nedrichards: 
> @Qwertie- in that case you should ideally apply the upstream change as a patch and then drop it once upstream makes a new release.

Appstream file is fixed in upstream. So the proper approach is to take that patch, add it to the manifest to patch the sources.

Then when a new release is out with the changes, just drop the patch.

**Comment by frhun** [comment]
Your pr is based on the wrong branch!
It should be based on new-pr.

**Comment by Mailaender** [comment]
Superseded by #3110.

**Pull request closed without merging**